.travis.yml

```diff
@@ -1,20 +1,30 @@
 ---
 language: csharp
 
-mono:
- - 4.6.1
- - latest # => "stable release"
- - weekly # => "latest commits"
+matrix:
+  include:      
+    - os: linux # Ubuntu 14.04
+      dist: trusty
+      sudo: required
+      dotnet: 1.0.0-preview2-003121
+      # Note that travis doesn't support dotnet cli rc4 via 'dotnet' yet,
+      # so we just use the old preview2 support to get things setup
+      # first, then manually add dotnet CLI RC4 during install.
+      env:
+        - CLI_VERSION=1.0.0-rc4-004771
+    - os: osx # OSX 10.11
+      osx_image: xcode7.3
+      dotnet: 1.0.0-preview2-003121
+      env:
+        - CLI_VERSION=1.0.0-rc4-004771
 
 install:
- - rvm install 2.2.3
- - gem install bundler
- - sudo apt-get update -y
+  - rvm install 2.2.3
+  - gem install bundler
+  - export DOTNET_INSTALL_DIR="$PWD/.dotnetcli"
+  - curl -sSL https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.sh | bash /dev/stdin --version "$CLI_VERSION" --install-dir "$DOTNET_INSTALL_DIR"
+  - export PATH="$DOTNET_INSTALL_DIR:$PATH"  
 
 script:
  - bundle && bundle exec rake
-
-matrix:
```

Comment:
I didn't think it's appropriate to test against future mono versions. I figured you probably just did it because it was easy and nice to have (but not super important to Expecto).